.github/workflows/release_publish.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           git remote add gh-origin https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
 
           npm version ${MAJOR_MINOR_SEMVER} --force -m "Bump version to %s"
-          git push gh-origin HEAD:master --tags
+          git push gh-origin --tags
```

Comment:
I don't think we need this line or the git remote add or the two git config commands above